style/main.css

```diff
@@ -178,7 +178,6 @@ nav{
     opacity: 1;
 }
 .back-to-top a{
-    position: absolute;
-    margin-bottom:3.5em;
+    position: fixed;

```

Comment:
Absolute positioning is not relative to the browser window, it is relative to the nearest non-static ancestor element.
You also don't need the margin-bottom here.